### DISCUSSION TITLE
why cannot search by tag？

### DISCUSSION THREAD

**Discussion by wonb168** [massCodeIO/massCode#338] [General]
this plugin can create snippet in masscode, but only can search by the title, cannot by tag I added in mass?

**Comment by LukasVolgger** [comment]
+Bump

Absolutely necessary feature!

**Comment by antonreshetov** [comment]
@wonb168 What plugin is this about? If the question is still about the application, then I don't see the point of searching by tags, as the user, in my opinion, needs to find a snippet. I also want to note that the search is not only by titles, but also by the content of the snippet.

**Reply by LukasVolgger** [reply]
@antonreshetov Thing is, the tags mostly contain information about the framework, language, environment, etc.  I also use Pieces for Developers and used SnipAway.  They support searches using titles, tags and content from a single source.  You simply have a much better chance of finding your snippets again if you have a lot.
